collections/eoxcloudless-s2-global-mosaic-10.yaml

```diff
@@ -12,7 +12,7 @@ Image: eoxcloudless-s2-global-mosaic-10/eoxcloudless-s2-global-mosaic-10.png
 Resolution: 10m
 GeographicalCoverage: Covering land surface areas between 180.0 West, 90.0 South
   and 180.0 East, 90.0 North
-TemporalAvailability: 2018, 2019, and 2020
+TemporalAvailability: ongoing
```

Comment:
If 2018, 2019,2020 are still available,
then   `TemporalAvailability:  2018 - Ongoing`   is abit more clearer

Reply:
Agreed